### ISSUE TITLE
Audio latency

### ISSUE THREAD

**Issue by mio-19** [utmapp/UTM#5076]
**Describe the issue**
Can see a very obvious audio latency

I used GuitarPro to play a song. The sound setting of Windows immediately shows a signal of sound playing, but the audio output of ipad has a very obvious latency

**Configuration (required)**
* UTM Version: 4.1.5
* OS Version: Windows 11 insider 25300
* Device Model: ipad Pro m1
* Is it jailbroken (name jailbreak used)? 
* How did you install UTM? TrollStore

**Crash log**
If the app crashed, you need a crash log. To get your crash log, open the Settings app and browse to `Privacy -> Analytics & Improvements -> Analytics Data` and find the latest entry for UTM. You should export the text and attach it here.

**Debug log**
For all issues, _including_ crashes, you should attach a debug log. Open UTM, and open the settings for the VM you wish to launch. Near the bottom of the configuration options (or at the top of the `QEMU` page) is `Debug Log`. Turn it on and save the VM. After you experience the issue, quit the VM and re-launch UTM. Open the VM settings again and select `Export Log...` and attach it here.

**Upload VM**
(Optional) If possible, upload the `config.plist` inside the `.utm`. If you do not have this, you can upload the entire `.utm` but note this contains your personal data. Since GitHub has an attachment size limit, you may want to upload to another service such as Google Drive. Link it here.

**Comment by mio-19** [comment]
It happens randomly. Possibly related to workload or display settings I changed recently

**Comment by osy** [comment]
Yeah, this is an known issue. I haven't been able to track down the cause yet, but it's probably somewhere in GST.

**Comment by osy** [comment]
Okay so the issue is that the first time your VM plays audio, there is a lot of work that is done setting up a gstreamer session and that's why there's latency. Subsequent audio tracks that play (provided that the sample rate and channels don't change) will keep using the same session and not have the issue. A workaround is to make sure you play some sound, wait a little bit, then everything will be synched up. A proper fix would involve identifying why setting up gstreamer takes so long and if we can improve the performance by reducing unneeded work. Alternatively we can "warm up" gstreamer by starting a session as soon as the VM is started. The downsides of this is that if the sample rate/channel changes, the setup will have to happen again.

**Comment by Andy312432** [comment]
Is it possible to set up it simontaniously when the VM boots as a workaround? Since most of the virtual machines won't play any sound during boot
